integration_tests/sdk/param_test.py

```diff
@@ -211,7 +228,58 @@ def test_trigger_flow_with_different_sql_param(client):
         # Verify the parameters were configured as expected.
         flow_runs = flow.list_runs()
         assert len(flow_runs) == 2
-        _check_param_vals(flow.fetch(flow_runs[1]["run_id"])._dag, expected_vals=["hotel_reviews"])
-        _check_param_vals(flow.latest()._dag, expected_vals=["customer_activity"])
+
+        historical_run = flow.fetch(flow_runs[1]["run_id"])
+        param_artifact = historical_run.artifact(name="table_name")
```

Comment:
I think the type of `param_artifact` is actually a `GenericArtifact` class instead of `ParamArtifact`. Could you confirm?